gitter/gitter-stream.js

```diff
@@ -0,0 +1,86 @@
+var https = require('https');
+const axios = require('axios');
+
+var roomId    = '5b92ffefd73408ce4fa74cbf'
+var token     = '49a37e04d62ff8583f7c31e8eea9c55f2e0ed3da';
+var heartbeat = " \n";
+
+var options = {
+  hostname: 'stream.gitter.im',
+  port:     443,
+  path:     '/v1/rooms/' + roomId + '/chatMessages',
+  method:   'GET',
+  headers:  {'Authorization': 'Bearer ' + token}
+};
+
+var req = https.request(options, function(res) {
+  res.on('data', function(chunk) {
+    var msg = chunk.toString();
+    userCheck(msg);
+    if (msg !== heartbeat) console.log('Message: ' + msg);
+  });
+});
+
+req.on('error', function(e) {
+  console.log('Something went wrong: ' + e.message);
+});
+
+userCheck = (msg) => {
+  const start = JSON.parse(msg);
+  console.log(msg);
+  
+  let userID, messageID, username, content, sent, currentMessageTime, globalTimeBenchMark;
+  messageID = start.id;
+  userID = start.fromUser.id;
+  username = start.fromUser.id;
+  sent = start.sent;
+  getAllMessages(userID);
+}
+
+getYesterday = () => {
+  let yesterday = new Date();
+  yesterday.setDate(yesterday.getDate()-1);
+  return yesterday;
+}
+
+getAllMessages = async (userId) => {
+  const yesterday = getYesterday();  
+  await getMessagesRequest(userId, yesterday);
+}
+
+getMessagesRequest = async (userId, yesterday) => {
+  const url = `https://api.gitter.im/v1/rooms/5b92ffefd73408ce4fa74cbf/chatMessages?access_token=49a37e04d62ff8583f7c31e8eea9c55f2e0ed3da`;
```

Comment:
access token on Github is not a good idea

Reply:
fixed